config/locales/idv/en.yml

```
@@ -185,7 +185,7 @@ en:
         review: Re-enter your %{app_name} password to protect your data
     troubleshooting:
       headings:
-        are_you_near: 'Are you near Washington, D.C.?'
+        are_you_near: 'Are you located near Washington, D.C.<wbr/>or Baltimore, MD?'
```

Couple observations:

* The `<wbr />` doesn't appear to create any space on its own in situations where there might not need to be a line break, so I think we should include one:

```suggestion
        are_you_near: 'Are you located near Washington, D.C. <wbr/>or Baltimore, MD?'
```

* Anecdotally, `<wbr />` doesn't work quite as I'd expect if the idea is to keep "or Baltimore, MD?" as a single continuous piece of text. Maybe we ought to be using non-breaking space characters instead?
   * Codepen demo: https://codepen.io/aduth/pen/oNdEZdm
   * Prior art: https://github.com/18F/identity-idp/blob/7415a07/app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx#L13-L38

---

> This is interesting. My understanding is that it prevents the `try_again` portion of the message from wrapping to a new line. Maybe this was solving a design issue? I can't infer much of the reasoning behind it from the blame. That approach also does similar string manipulation. I wonder if the helper belongs alongside `formatHtml`...
> 
> Still, we use lots of HTML throughout the locales yaml files. In the spirit of using a single approach, maybe that prior attempt should be changed to use explicit `&nbsp;` inside the yaml content? And remove the post-processing helper? Although that might miss something that the usage of the [unicode](https://github.com/18F/identity-idp/blob/7415a07/app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx#L17) representation of the non-breaking space was trying to solve...

Yes, as the person who had implemented that, I can confirm it was for design / legibility reasons to explicitly control the line breaks.

I don't have a strong opinion whether that should happen in the YAML or as a post-processing step, but I have a soft preference for the latter, to keep the YAML concerned with just the content, and any presentational concerns in the realm of the application code.